src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -25,15 +28,29 @@ export default App.extend({
   },
   onStart({ patient }) {
     this.patient = patient;
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    this.canManagePatients = currentUser.can('patients:manage');
```

Comment:
Based on what's in the backend, `patients:manage` is needed in order to update the patient's status.

There is also a `patients:owned:manage` permission in the backend. But it doesn't look like it's used for anything.